src/OrchardCore/OrchardCore.Navigation.Core/NavigationManager.cs

```diff
@@ -37,7 +37,7 @@ IAuthorizationService authorizationService
             _authorizationService = authorizationService;
         }
 
-        public async Task<IEnumerable<MenuItem>> BuildMenuAsync(string name, ActionContext actionContext)
+        public async Task<IEnumerable<MenuItem>> BuildMenuAsync(string name, ActionContext actionContext, string adminUrlPrefix)
```

Comment:
Why?

Reply:
Ignore this. I have changed the way to get it from DI.